h2/src/main/org/h2/table/InformationSchemaTable.java

```diff
@@ -109,7 +109,9 @@ public final class InformationSchemaTable extends MetaTable {
 
     private static final int REFERENTIAL_CONSTRAINTS = PARAMETERS + 1;
 
-    private static final int ROUTINES = REFERENTIAL_CONSTRAINTS + 1;
+    private static final int RESTORE_POINTS = REFERENTIAL_CONSTRAINTS + 1;
```

Comment:
This is not a standard view, it needs to be placed below under `Extensions` comment.